### PR TITLE
Task-57767: Decoding documents' names with special characters in the Documents application

### PR DESCRIPTION
**Pull request by sofyenne**
ISSUE : When the user launches the documents application, an encoded list of document names with special characters is displayed.
FIX : Decode the document name .

### CODE REVIEW

File: documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue [hunk 1]
```diff
@@ -117,6 +117,9 @@ export default {
     },
   }),
   computed: {
+    title(){
+     this.file.name=decodeURI(this.file.name);
+    },
```
Comment (boubaker) [inline documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue]:
```suggestion
    title() {
      return decodeURI(this.file.name);
    },
```

Comment (sofyenne) [reply documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue]: Done !

**Review by boubaker — DISMISSED** [review]

File: documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue [hunk 2]
```diff
@@ -238,6 +241,7 @@ export default {
     openPreview() {
       this.loading = true;
       if (this.file && this.file.folder){
+        this.file.name = decodeURI(this.file.name);
```
Comment (boubaker) [inline documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue]: we should avoid changing retrieved information and keep it as it is. BTW, you can add a dedicated data variable with different name such as `fileName` and assing into it the decoded value.

**Comment by boubaker** [comment]
@sofyenne I can't reproduce the problem anymore, can you check again please

File: documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue (hunk 2 above)
Comment (sofyenne) [reply documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue]: i leave this change i find is not necessary after a test of fixed behavior

**Review by boubaker — APPROVED** [review]

**Pull request merged**